website/thaliawebsite/views.py

```diff
@@ -19,3 +27,11 @@ def dispatch(self, request, *args, **kwargs) -> HttpResponse:
         if not request.user.is_superuser:
             return HttpResponseForbidden("This is not for you")
         raise Exception("Test exception")
+
+
+def short_url(request, slug):
+    link = get_object_or_404(ShortLink, key=slug)
+    if not link.immediate:
+        return render(request, "shortlink.html", {"link_name": link.key, "link_url": link.url})
+
+    return HttpResponseRedirect(link.url)
```

Comment:
This could just as well be a class-based view. Just like everything else in this file.